docs/src/_guide/troubleshooting.md

```diff
@@ -68,6 +68,20 @@ meltano            | DEBUG Invoking: ['demo-project/.meltano/loaders/target-json
 meltano            | DEBUG Env: {'MELTANO_EXTRACTOR_NAME': 'tap-gitlab', 'MELTANO_EXTRACTOR_NAMESPACE': 'tap_gitlab', 'MELTANO_EXTRACT_API_URL': 'https://gitlab.com', 'MELTANO_EXTRACT_PRIVATE_TOKEN': '', 'MELTANO_EXTRACT_GROUPS': '', 'MELTANO_EXTRACT_PROJECTS': 'meltano/meltano', 'MELTANO_EXTRACT_ULTIMATE_LICENSE': 'False', 'MELTANO_EXTRACT_START_DATE': '2021-03-01', 'TAP_GITLAB_API_URL': 'https://gitlab.com', 'GITLAB_API_TOKEN': '', 'GITLAB_API_GROUPS': '', 'GITLAB_API_PROJECTS': 'meltano/meltano', 'GITLAB_API_ULTIMATE_LICENSE': 'False', 'GITLAB_API_START_DATE': '2021-03-01', 'TARGET_JSONL_DESTINATION_PATH': 'output', 'TARGET_JSONL_DO_TIMESTAMP_FILE': 'False'}
 ```
 
+## No Plugin Settings Defined
+
+To configure a plugin that does not already have settings defined, we recommend first adding a `settings:` entry under the plugin definition. You may need to consult the plugin's documentation to determine what settings are available.
+
+Related reference information and guides:
+
+- [Overriding discoverable plugin properties](https://docs.meltano.com/guide/configuration#overriding-discoverable-plugin-properties)
+- [Plugin `settings` syntax reference](https://docs.meltano.com/reference/plugin-definition-syntax#settings)
+- [Custom settings guide](https://docs.meltano.com/guide/configuration#custom-settings)
+
+After adding definitions for the plugin's settings, you may then configure those settings as usual with the `--interactive` option in `meltano config`.
+
+You may also contribute back to the community by publishing the settings to [MeltanoHub](https://hub.meltano.com) in the form of a new pull request.
+
```

Comment:
@aaronsteers I won't approve this since it's my own PR, but this doc update looks good to me.

Reply:
Thanks, @WillDaSilva ! @tayloramurphy is out at Coalesce so let's go ahead and merge. 🚀